EPNTAP.tex

```diff
@@ -2376,8 +2378,8 @@ \section{EPNCore Table}
 target\_primary\_hemisphere&Text&&Primary observed hemisphere&\ucd{meta.id;obs.field}\\
 target\_secondary\_hemisphere&Text&&Secondary observed hemisphere&\ucd{meta.id;obs.field}\\
 platesc&Float&arcsec/pix&pixel angular size or platescale (on sky only)&\ucd{instr.scale}\\
-detector\_orientation&Float&deg&Position angle of image y axis (on sky only)&\ucd{pos.posAng}\\
-mesaurement\_unit&Text&&Physical unit, same as Bunit in fits&\ucd{pos.posAng}\\
+orientation&Float&deg&Position angle of image y axis (on sky only)&\ucd{pos.posAng}\\
+measurement\_unit&Text&&Physical unit, same as Bunit in fits&\ucd{pos.posAng}\\
```

Comment:
The UCD for measurement_unit is listed as pos.posAng.  That doesn't look right.

Reply:
indeed…

Reply:
fixed here